### PR TITLE
Route to next page if no joint application has been selected.

### PR DESCRIPTION
**Pull request by claytn**

### CODE REVIEW

File: src/components/apply/ApplicantForm.js [hunk 1]
```diff
@@ -20,25 +22,42 @@ const headingStyle = {
...
-            label="Add joint applicant to my membership application"
-            labelStyle={{color: colors.basic}}
-            iconStyle={{fill: colors.basic, marginLeft: 10}}
-        />
-      </Flex>
-      <center><ContinueButton title='KEEP GOING' buttonProps={{type: 'submit'}} style={{ marginTop: 10 }}/></center>
-    </form>
-  </div>
+    <form onSubmit={handleSubmit((values) =>{
+      console.log(values);
```
Comment (jimmyhmiller) [inline src/components/apply/ApplicantForm.js]: Remove this console statement

File: src/components/apply/ApplicantForm.js [hunk 2]
```diff
@@ -20,25 +22,42 @@ const headingStyle = {
...
-            component={Checkbox}
-            label="Add joint applicant to my membership application"
-            labelStyle={{color: colors.basic}}
-            iconStyle={{fill: colors.basic, marginLeft: 10}}
-        />
-      </Flex>
-      <center><ContinueButton title='KEEP GOING' buttonProps={{type: 'submit'}} style={{ marginTop: 10 }}/></center>
-    </form>
-  </div>
+    <form onSubmit={handleSubmit((values) =>{
```
Comment (jimmyhmiller) [inline src/components/apply/ApplicantForm.js]: At the very least, I want to see this moved out into a named function. But we should consider handling this at a different level. Triggering a redux action here and handing the redirect elsewhere.

File: src/components/apply/ApplicantForm.js [hunk 3]
```diff
@@ -20,25 +22,42 @@ const headingStyle = {
...
+      console.log(values);
+      if(!addJointApplicant){
```
Comment (jimmyhmiller) [inline src/components/apply/ApplicantForm.js]: Why not get this from values?

**Review by jimmyhmiller — CHANGES_REQUESTED** [review]

File: src/components/apply/ApplicantForm.js (hunk 2 above)
Comment (claytn) [reply src/components/apply/ApplicantForm.js]: What would be the perks of handling routing in redux vs using withRouter inside of the component?

Comment (jimmyhmiller) [reply src/components/apply/ApplicantForm.js]: Imagine that we want to submit to the backend at this point to allow people to save their application and resume. Or imagine that we want to record some analytics about how long it took the user to fill out our form. Or imagine that we need to do an identity verification.

Basically, if this turns into an asynchronous process in anyway, it becomes really beneficially to use some of the tools the redux ecosystem offers us for handling those.

Not saying this needs to be converted to that. As long as it is pulled out into a named function I will approve it. But it is something to think about.

**Comment by claytn** [comment]
I made the changes you requested, but I am going to work on moving the routing to redux later today. For the time being the formSubmit is done in a HOF.

**Review by jimmyhmiller — APPROVED** [review]

**Pull request merged**